packaging/packager_rpm.sh

```diff
@@ -12,30 +12,33 @@ dnf -y install python3 python3-coverage
 dnf -y install python3-pillow
 # pip dependencies (for dependencies not available as RPM)
 dnf -y install gcc libX11-devel libXtst-devel python3-devel libpng-devel python3-pip redhat-rpm-config
+
 pip3 install --upgrade pip
+pip3 install setuptools setuptools-rust
+pip3 install pillow==10.4.0
+
 # contour, template, feature, cascade, text matching
 dnf -y install python3-numpy python3-opencv
+pip3 install numpy opencv-python
+
 # text matching
 dnf -y install tesseract tesseract-devel
 dnf -y install gcc-c++
-pip3 install pytesseract==0.3.4 tesserocr==2.5.1
+pip3 install pytesseract==0.3.10 tesserocr==2.7.0
+
 # deep learning
-pip3 install torch==1.8.1 torchvision==0.9.1
+pip3 install torch==2.2.0 torchvision==0.17.0
+
 # screen controlling
-if (( distro_version <= 32 )); then
-    pip3 install autopy==4.0.0
-else
-    export DISABLE_AUTOPY=1
-fi
+export DISABLE_AUTOPY=1
```

Comment:
There is no need to remove the conditional here, we can keep the dynamic nature of this for backwards portability with older Fedora distros.

Reply:
Okay I'll put this one back